README.md

```diff
@@ -1,95 +1,69 @@
 # Ably Documentation Overview
 
-[Ably](https://www.ably.io) is a hugely scalable, superfast and secure hosted realtime messaging service for web-enabled devices.
+[Ably](https://www.ably.io) is a scalable, fast, and secure hosted realtime messaging service for web-enabled devices.
 
-This Git repository contains most of the [Ably API documentation](https://www.ably.io/documentation) that resides at <https://www.ably.io/documentation> in [Textile](redcloth.org/textile) format.  [Nanoc](http://nanoc.stoneship.org/) is used to build the static site from the `master` branch.
+The static site generated from this repository is hosted at <http://docs.ably.io> and deployed automatically when the `master` branch is updated.
 
-The Ably documentation was intentionally created as a public repository using a simple text based markup language so that:
+We frequently publish updates from this repository so is typically more up to date than the official Ably documentation which can be found at <https://www.ably.io/documentation>.
 
-* It can be maintained by Ably staff and developers.
-* 3rd parties can fork the repository, and submit pull requests to improve the documentation.  We welcome the community's suggestions and input.
-* This documentation is later merged upstream to the [Ably primary website documentation](https://www.ably.io/documentation) with each website release.
+This Git repository uses the [Textile](https://redcloth.org/textile) format with [Nanoc](http://nanoc.stoneship.org/) used to build the static site from the `master` branch.
```

Comment:
I think it's important we include a reference to the documentation on how to write content in `content/client-lib-development-guide/documentation-formatting-guide.textile` as anyone who wants to contribute, would need to read that to get going.